manifests/server/config.pp

```diff
@@ -147,6 +147,18 @@
                   Exec['puppet_server_config-create_ssl_dir'],
                   ],
     }
+  } elsif $::puppet::server::ca_crl_sync {
+    # If not a ca AND sync the crl from the ca master
+    if $::server_facts['servername'] != undef {
+      file { $::puppet::server::ssl_ca_crl:
+        ensure  => file,
+        owner   => $::puppet::server::user,
+        group   => $::puppet::server::group,
+        mode    => '0644',
+        content => file($::settings::cacrl, $::settings::hostcrl, '/dev/null'),
+        require => Package['pe-puppetserver'],
```

Comment:
This was errantly included from other code. Cleaned up... ( I need to stop writing code in the middle of the night)